erpnext/accounts/report/deferred_revenue_and_expense/deferred_revenue_and_expense.py

```
@@ -289,7 +314,11 @@ def get_invoices(self):
 			.join(inv)
 			.on(inv.name == inv_item.parent)
 			.left_join(gle)
-			.on((inv_item.name == gle.voucher_detail_no) & (deferred_account_field == gle.account))
+			.on(
+				(inv_item.name == gle.voucher_detail_no)
+				& (deferred_account_field == gle.account)
```

Pylint (3.3.7)

[error] Possibly using variable 'deferred_account_field' before assignment

(E0606)